docs/user/estimate_migration.rst

```diff
@@ -0,0 +1,160 @@
+====================
+Migration Estimation
+====================
+
+Quick Start Guide
+-----------------
+
+1. Add credentials for source and destination cloud to `discover.yaml` file,
```

Comment:
Please use **double** backticks to highlight files, and bash commands, single backticks don't work here as in markdown see [sphinx documentation](http://www.sphinx-doc.org/en/stable/rest.html#inline-markup).